documentation/quarkus.asciidoc

```diff
@@ -11,21 +11,21 @@ Additional things like extensions will be available on the  https://github.com/d
 
 Depending on your intention of reading this document, you might be more interested in the following chapters:
 
-* If you are completely new to Quarkus, you may be interested in the link:quarkus.asciidoc#pros[pros] and link:quarkus.asciidoc#cons[cons] of Quarkus. Also, take a look at the https://quarkus.io[official Quarkus website]. You might also be interested in the features that https://www.graalvm.org/[GraalVM] offers.
+* If you are completely new to Quarkus, you may be interested in the link:https://devonfw.com/website/pages/docs/devon4j.wiki_quarkus.asciidoc_pros_2.html[pros] and link:https://devonfw.com/website/pages/docs/devon4j.wiki_quarkus.asciidoc_cons_4.html[cons] of Quarkus. Also, take a look at the https://quarkus.io[official Quarkus website]. You might also be interested in the features that https://www.graalvm.org/[GraalVM] offers.
```

Comment:
same here.
Most probably the desired link should have been `xref:pros[pros]` instead.